### ISSUE TITLE
urtf model

### ISSUE THREAD

**Issue by Jyb-mouse** [koide3/iridescence#137]
Does it support loading URTF models？

**Comment by koide3** [comment]
No. But, there is general mesh model IO that may be useful for your purpose: https://github.com/koide3/iridescence/blob/master/src/example/ext_model_viewer.cpp
